### PR TITLE
Temporary workaround for incorrect Blazor CLI package metadata generation

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
This is to fix `blazor serve`.

The problem was that previously we relied on referencing parts of ASP.NET Core runtime as packages, but since it will ship as shared framework only, we can no longer do that. We now need to use `<FrameworkReference>` to reference ASP.NET Core, but the build system doesn't yet support that for this kind of package.

@natemcmaster is working on adding `<FrameworkReference>` support, so this PR is intended as a *very temporary* workaround to unlock verification that the rest works end-to-end.

### CODE REVIEW

File: src/Components/Blazor/Cli/src/Microsoft.AspNetCore.Blazor.Cli.csproj [hunk 1]
```diff
@@ -21,4 +21,26 @@
...
+      <NuspecProperties>
+        outdir=$(OutDir);
+        componentsrootdir=..\..\..\;
+        blazorversion=$(PackageVersion);
+        aspnetcoreversion=$(ComponentsPackageVersion);
+        repositorycommit=$(RepositoryCommit);</NuspecProperties>
+    </PropertyGroup>
+    <ItemGroup>
+      <!-- Also temporarily override RuntimeFramework manually with hardcoded version until we can use <FrameworkReference> -->
+      <RuntimeFramework Include="Microsoft.AspNetCore.App" Version="3.0.0-preview-19064-0344" />
```
Comment (SteveSandersonMS) [inline src/Components/Blazor/Cli/src/Microsoft.AspNetCore.Blazor.Cli.csproj]: This is the most troublesome part of the hack. I don't know if there's any way to look up the correct version dynamically.

@natemcmaster Does the CI environment inject any value we can use for this?

Comment (rynowak) [reply src/Components/Blazor/Cli/src/Microsoft.AspNetCore.Blazor.Cli.csproj]: A bunch of places use `$(MicrosoftAspNetCoreAppPackageVersion)` (like templating)

**Review by rynowak — APPROVED** [review]
Looks fine to me, but someone like @natemcmaster or @pranavkm should have a look

File: src/Components/Blazor/Cli/src/Microsoft.AspNetCore.Blazor.Cli.csproj (hunk 1 above)
Comment (SteveSandersonMS) [reply src/Components/Blazor/Cli/src/Microsoft.AspNetCore.Blazor.Cli.csproj]: It looks like there isn't a repo-wide `$(MicrosoftAspNetCoreAppPackageVersion)` property, but the places that use it create their own, setting the value to `$(PackageVersion)`, so I'm now doing that.

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**